pkgs/by-name/km/kminion/package.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, buildGoModule
+}:
+
+buildGoModule rec {
+  pname = "kafka-minion";
+  version = "2.2.7";
+
+  src = fetchFromGitHub {
+    owner = "redpanda-data";
+    repo = "kminion";
+    rev = "0c90d4301ed4600d1aaf3345b6f16587d2f282fc";
+    hash = "sha256-CWjX46Sfc9Xj+R7+CZeMuTY0iUStzyZXI4FotwqR44M=";
+  };
+
+  vendorHash = "sha256-6yfQVoY/bHMA4s0IN5ltnQdHWnE3kIKza36uEcGa11U=";
+
+  doCheck = false;
+
+  passthru.updateScript = ./update.sh;
+
```

Comment:
```suggestion
```
The update bot will already try nix-update on the package

Reply:
I think it only gets automatically updated from repology by default\[1\], and since it would only be on repology from nixpkgs (when this merges)\[2\], I don't think it  would be automatically updated, so I'm pretty sure we should keep this to get updates from github releases (correct me if I'm wrong, I don't have a great understanding of the bot).

\[1\]: https://github.com/NixOS/nixpkgs/blob/f98a3ccfa6faa5da98aa361835f005fc9a148f5d/doc/stdenv/stdenv.chapter.md?plain=1#L513
\[2\]: https://repology.org/projects/?search=kminion

Reply:
Sounds correct to me, there are no other sources of truth for it for repology.